src/main/java/uk/gov/companieshouse/companyprofile/delta/mapper/CompanyProfileEnumMapper.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.companieshouse.companyprofile.delta.mapper;
+
+import org.mapstruct.*;
+import uk.gov.companieshouse.api.company.Accounts;
+import uk.gov.companieshouse.api.company.CompanyProfile;
+import uk.gov.companieshouse.api.company.Data;
+import uk.gov.companieshouse.api.company.LastAccounts;
+import uk.gov.companieshouse.api.delta.BooleanFlag;
+import uk.gov.companieshouse.api.delta.CompanyDelta;
+
+
+import javax.validation.constraints.Null;
+import java.util.HashMap;
+
```

Comment:
This class isn't used anymore now all the Mappings are in the main mapper class right? If it isn't used we should delete it.